build/storage/tests/it/ptf_tests/system_tools/errors.py

```diff
@@ -0,0 +1,21 @@
+"""System tools errors"""
+
+
+class ContainerNotRunningException(Exception):
+    """Container is not currently running"""
+
+
+class CommandException(Exception):
+    """Custom Exception raises if error occurs during command execution"""
+
+
+class VirtualizationException(Exception):
+    """Virtualization is not setting properly"""
+
+
+class DependenciesException(Exception):
+    """Dependency is not setting properly"""
+
+
+class BusyPortException(Exception):
+    """There is process in port"""
```

Comment:
The requested port is occupied by another process

Reply:
changed